core/src/test/java/org/elasticsearch/index/query/plugin/DummyQueryParserPlugin.java

```diff
@@ -83,8 +86,8 @@ protected int doHashCode() {
         }
 
         @Override
-        public String getWriteableName() {
-            return NAME;
+        protected boolean doEquals(DummyQueryBuilder other) {
+            return true;
```

Comment:
Yep. I think there were some other changes at some point but they weren't needed, I'll revert this file